substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfInfoSectionImpl.java

```diff
@@ -326,10 +1326,207 @@ public int writeAttrString(String value, byte[] buffer, int p) {
         }
     }
 
+    public int writeAttrAccessibility(int modifiers, byte[] buffer, int p) {
+        byte access;
+        if (Modifier.isPublic(modifiers)) {
+            access = DwarfDebugInfo.DW_ACCESS_public;
+        } else if (Modifier.isProtected(modifiers)) {
+            access = DwarfDebugInfo.DW_ACCESS_protected;
+        } else if (Modifier.isPrivate(modifiers)) {
+            access = DwarfDebugInfo.DW_ACCESS_private;
+        } else {
+            /* Actually package private -- make it public for now. */
+            access = DwarfDebugInfo.DW_ACCESS_public;
+        }
+        return writeAttrData1(access, buffer, p);
+    }
+
+    public int writeIndirectOopConversionExpression(boolean isHub, byte[] buffer, int p) {
+        int pos = p;
+        /*
+         * The conversion rules are different depending on whether they apply to the hub class or
```

Comment:
Now that we have all this info in the JavaDoc of com.oracle.svm.core.heap.ReferenceAccess please refer to it here instead of replicating.

Reply:
Done